docs/CORTX-S3 Server Quick Start Guide.md

```diff
@@ -44,6 +44,7 @@ This guide provides a step-by-step walkthrough for getting you CORTX-S3 Server r
       * To check if pip is installed, use: `$ pip --version`
       * To check if epel is installed, use: `$ yum repolist`. If epel was installed, you'll see it in the output list. If not enable it using :`$ yum --enablerepo=extras install epel-release`.
       * To install pip use: `$ yum install python-pip`
+      * A specific version is needed: `$ pip install pip==20.3.3`

```

Comment:
Thanks for being proactive. better to have line 46/47 merged

Reply:
Hi @nileshgovande, I'm not sure if I understand the error correctly, it seems that pip on Python 2 was causing the error, and python-pip is pip3?

Reply:
Did a check and seems like pip3 is under python3-pip package, but I couldn't find pip version 20.3.3 using yum, or are you referring to something like `yum install python-pip && pip install pip==20.3.3`?

Reply:
Will close this since Saumya is handling it [#801](https://github.com/Seagate/cortx-s3server/pull/801).